content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+```json
+{
+  "name": "VMware create three tiers",
+  "description": "Snapshot-local-cloud",
+  "protections": [
+    {
+      "type": "SNAPSHOT",
+      "schedules": [
+        {
+          "scheduleId": 1,
+          "name": "Array_Snapshot_1",
+          "namePattern": {
+            "format": "Array_Snapshot_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "DAYS",
+            "value": 1
+          },
+          "schedule": {
+            "recurrence": "HOURLY",
+            "repeatInterval": {
+              "every": 4
+            },
+            "activeTime": {
+              "activeFromTime": "00:00",
+              "activeUntilTime": "23:59"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "type": "BACKUP",
+      "protectionStoreId": "<onprem-protection-store-id>",
+      "schedules": [
+        {
+          "scheduleId": 2,
+          "name": "On-Premises_Protection_Store_2",
+          "sourceProtectionScheduleId": 1,
+          "namePattern": {
+            "format": "On-Premises_Protection_Store_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "DAYS",
+            "value": 3
+          },
+          "schedule": {
+            "recurrence": "DAILY",
+            "repeatInterval": {
+              "every": 1
+            },
+            "startTime": "00:00"
+          }
+        }
+      ]
+    },
+    {
+      "type": "CLOUD_BACKUP",
+      "protectionStoreId": "<cloud-protection-stores-id>",
+      "schedules": [
+        {
+          "scheduleId": 3,
+          "name": "HPE_Cloud_Protection_Store_3",
+          "sourceProtectionScheduleId": 2,
+          "namePattern": {
+            "format": "HPE_Cloud_Protection_Store_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "WEEKS",
+            "value": 1
+          },
+          "schedule": {
+            "recurrence": "DAILY",
+            "repeatInterval": {
+              "every": 2
+            },
+            "startTime": "00:00"
+          }
+        }
+      ]
+    }
+  ],
+  "applicationType": "VMWARE"
+}
+```
+
+> *The above figure shows JSON structure for request body of `POST /backup-recovery/v1beta1/protection-policies` for the creation of protection-policy as shown by the example in this section.*
+
+6. Finally, I created of the protection policies using the GreenLake API for Backup and Recovery `POST /backup-recovery/v1beta1/protection-policies`, and I used the above JSON structure in the request body. This is a special POST API execution where the response is returned immediately. The response of this API contained the body of JSON structure that will be useful to identify the protection-jobs such as the `“<protection-policies-id>”`.
```

Comment:
I think you mean "Finally, I created the protection policies" ("of" doesn't belong in here).